_posts/2016-07-16-leaflet-1.0-rc2.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Announcing Leaflet 1.0-rc2
+description: Leaflet 1.0 Release Candidate 2 is out
+author: Yohan Boniface
+authorsite: http://yohanboniface.me
+---
+
+Here comes the second release candidate for Leaflet 1.0,  the more stable Leaflet version ever!
+
+
+Leaflet 1.0-rc2 is mainly bug fixing, with two notable exceptions. The full list of changes is in the [1.0.0-rc2 changelog](https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md#10-rc2-july-18-2016) as usual.
+
+
+### Notable changes
+
+* Event refactoring: thanks to an initial work from [@fab1an](https://github.com/fab1an), coordinated by [@perliedman](https://github.com/perliedman), the events are greatly optimized: they are faster, and consume less memory.
+
+* New `L.Tooltip` class: started as a port of Leaflet.Label plugin to Leaflet core, and then reworked, this new class allows to display small tooltips attached to map features. The API is very similar to what Leaflet.Label was (with some exceptions in options naming and default values), so the upgrade path from Leaflet.Label to `L.Tooltip` should be straightforward, and we encourage you to do so while upgrading your Leaflet to 1.0.
```

Comment:
Added to https://github.com/Leaflet/Leaflet/issues/4732, and I'll do a PR to Leaflet.Label for that.